src/components/Tabs.react.js

```diff
@@ -0,0 +1,204 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const greyColor = '#d6d6d6';
+const offWhiteColor = '#f9f9f9';
+const primaryColor = '#1975FA';
+
+const EnhancedTab = ({
+  index,
+  label,
+  selected,
+  className,
+  style,
+  selectedClassName,
+  selectedStyle,
+  selectHandler
+}) => {
+  return (
+    <div
+      className={
+        selected
+          ? `${className || ''} ${selectedClassName || ''} tab tab--selected`
+          : `${className || ''} tab`
+      }
+      style={selected ? { ...style, ...selectedStyle } : style}
+      onClick={() => {
+        selectHandler(index);
+      }}
+    >
+      <span>{label}</span>
+      <style jsx>{`
+        .tab {
+          display: inline-block;
+          background-color: ${offWhiteColor};
+          border: 1px solid ${greyColor};
+          padding: 20px;
+          transition: background-color, color 200ms;
+          font-family: 'system-ui';
+          width: 100%;
+          box-sizing: border-box;
+        }
+        .tab:hover {
+          cursor: pointer;
+        }
+        .tab--selected {
+          border-top: 2px solid ${primaryColor};
+          border-bottom: none;
+          color: black;
+        }
+
+        @media screen and (min-width: 1000px) {
+          .tab {
+            width: auto;
+          }
+        }
+      `}</style>
+    </div>
+  );
+};
+
+export default class Tabs extends Component {
```

Comment:
Let's add a docstring above this component - these docstrings will get picked up by the Dash component toolchain and will become available in the Python `help(dcc.Tabs)` context